config/plugins/visualizations/charts/config/charts.xml

```diff
@@ -5,6 +5,7 @@
         <data_source>
             <model_class>HistoryDatasetAssociation</model_class>
             <test type="isinstance" test_attr="datatype" result_type="datatype">tabular.Tabular</test>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">tabular.CSV</test>
```

Comment:
The visualization framework calls the registry to lookup the datatype class by name/string.

Generally the superclasses have had less in common with each other as far as information contained than their format or methods for sniffing AFAIK (e.g. Binary, Text, etc.) with some exceptions (Interval).